Minigin/Scene.cpp

```diff
@@ -1,6 +1,8 @@
 #include "Scene.h"
 #include "GameObject.h"
 
+#include <algorithm>
```

Comment:
This line was not required in MSVC because of a transitive include; it is required for building on gcc (tested with 13.2.1). It could be useful to add [IWYU](https://include-what-you-use.org/) to the CMake tooling to find more of these.

Reply:
Correct